sklearn/datasets/base.py

```diff
@@ -255,6 +255,12 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+    ----------
+    return_X_y : boolean, default=False.
+        If true, you will get (X, y), else you will get Bunch.
+
+
     Returns
     -------
     data : Bunch
```

Comment:
I suppose something like ``Bunch or tuple`` and also mention the default while explaining that it depends on ``return_X_y`` in the next line would be a good option. Just a suggestion.